splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -315,6 +335,9 @@ protected void createDictionaryTables(Properties params,
         //create the Statistics tables
         createStatisticsTables(tc);
 
+        // TODO - this needs to be included into an upgrade script (JY)
```

Comment:
is //TODO comment here still needed, I see that there is logic added to upgradeSystables() for the new system table created?